internal/header/accept_features.go

```diff
@@ -0,0 +1,70 @@
+/* Copyright 2019 Freerware
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package header
+
+// AcceptFeatures represents the Accept-Features header.
+//
+// The Accept-Features request header can be used by a user agent to
+// give information about the presence or absence of certain features in
+// the feature set of the current request.  Servers can use this
+// information when running a remote variant selection algorithm.
+type AcceptFeatures struct {
```

Comment:
can consider having `type AcceptFeatures []FeatureExpression` instead, as the struct simply has this single property.

Reply:
done!